src/lib/components/DatePicker/DatePicker.js

```diff
@@ -27,18 +40,37 @@ class DatePicker extends Component {
       id,
       name,
       placeholder,
+      disable,
+      disabledInput,
+      loading,
     } = this.props;
     const { selectedDate } = this.state;
+    // Remove this code when this issue is solved -> https://github.com/arfedulov/semantic-ui-calendar-react/issues/202
+    // and update `initialDate={initialDateChecked}` to `initialDate={initialDate}`
+    //----------------------------------------------------------------------------
+    let firstAvailableDay = DateTime.local();
```

Comment:
I have noticed that we convert `toShortDate(firstAvailableDay)` multiple times below, I am not sure it is needed. Can you double check?
I would just do it at the very end if you need to convert it to a string.

It will be also great if you encapsulate all this in a function, e.g. `const initialDateChecked = getInitialDate(...)`

Reply:
We have to call `toShortDate(firstAvailableDay)` to check if it's included in the `disable` since this is an array of strings, but we can encapsulate it in a function and remove 1 unnecesary `toShortDate(firstAvailableDay)`. Thanks!